H5P.BranchedVideo/branched-video.js

```diff
@@ -47,6 +47,69 @@ H5P.BranchedVideo = (function ($) {
       this.currentTime = 0.00;
       this.watchedTime = 0.00;
       this.nodes = [];  // array of node objects
+
+      this.playedTime = [[0,0]];
+
+      this.getPlayedTime = function(){
+        return this.playedTime;
+      }
+
+      this.updatePlayedTime = function(time){
+        var roundedTime = Math.round( time * 10 ) / 10; // make it 1 decimal
+        var arrayOfPlayedTime = this.playedTime;
+        var index = getIndex();
+        updateArray();
+        this.playedTime = arrayOfPlayedTime;
+
+        function getIndex(){
+          var length = arrayOfPlayedTime.length;
+          for (var i = 0 ; i<length; i++){
```

Comment:
Please put a space before and after `<`. Also keep your semicolon spacing consistent. I recommend not adding a space before semicolons.